packages/subgraph/schema.graphql

```diff
@@ -1,10 +1,70 @@
-# Types
-type DaoPlugin @entity {
-  "OSX related data"
-  id: ID! # psp installationId
-  dao: Bytes!
+interface IPlugin {
+  id: ID! # plugin address
```

Comment:
```suggestion
  id: ID! # PluginEntityID
```

Let's write the precise ID name everywhere. By checking the `generatePluginEntityId` function, one can then clearly see that it is the plugin address.
```ts
export function generatePluginEntityId(plugin: Address): string {
  return generateEntityIdFromAddress(plugin);
}
```